desc/objectives/_stability.py

```diff
@@ -184,7 +182,9 @@ def compute(self, R_lmn, Z_lmn, L_lmn, p_l, i_l, c_l, Psi, **kwargs):
             self._iota,
             self._current,
         )
-        return self._shift_scale(compress(self.grid, data["D_Mercier"]))
+        f = compress(self.grid, data["D_Mercier"], surface_label="rho")
+        w = compress(self.grid, self.grid.weights, surface_label="rho")
+        return self._shift_scale(f * w)
```

Comment:
In the `ToroidalCurrent` objective the radial weights are computed as follows:
```
I = compress(self.grid, data["current"], surface_label="rho")
w = compress(self.grid, self.grid.spacing[:, 0], surface_label="rho")
return self._shift_scale(I * w)
```
Why would it be any different for the Mercier objectives? I mean the difference between using `self.grid.weights` vs `self.grid.spacing[:, 0]`